docs/releases/2.0.txt

```diff
@@ -206,7 +206,7 @@ URLs
 Validators
 ~~~~~~~~~~
 
-* ...
+* The :class:`django.core.validators.EmailValidator` now uses the HTML5 email validation standard.
```

Comment:
Wrap documentation at 79 characters -- not sure I'd consider this a "minor feature" though.